lib/phi_attrs/phi_record.rb

```diff
@@ -71,6 +71,8 @@ def extend_phi_access(*methods)
       #   Foo.allow_phi!('user@example.com', 'viewing patient record')
       #
       def allow_phi!(user_id, reason)
+        return if user_id.blank? || reason.blank?
```

Comment:
Should we not raise an `ArgumentError` here? Quiet failures scare me.

Reply:
Sure. I think that's sensible.

Reply:
Updated